domains/alpha.json

```diff
@@ -4,8 +4,8 @@
         "email": "sweepalf@gmail.com"
     },
     "record": {
-        "A": ["217.174.245.249"],
-        "MX": ["hosts.is-a.dev"],
-        "TXT": ["v=spf1 a mx ip4:217.174.245.249 ~all"]
+        "CNAME": "yfaweb.pages.dev",
```

Comment:
Cannot have a CNAME in conjunction with other records.

Reply:
Resolved.